manifests/slave.pp

```diff
@@ -170,6 +176,16 @@
     }
   }
 
+  if $slave_args {
+    if is_array($slave_args) {
+      $_combined_slave_args = hiera_array('jenkins::slave::slave_args', $slave_args)
```

Comment:
@rtyler, fair enough. The docs can be very confusing. If I change line 181-182 to read more like:
```
$_real_slave_args = join($slave_args, ' ')
```
Would you be okay with that?